admin_guide/manage_rbac.adoc

```diff
@@ -762,8 +762,98 @@ To bind the new role to a user, run:
 $ oc adm policy add-role-to-user podview user2 --role-namespace=bob-project -n bob-project
 ----
 
+[[creating-cluster-role]]
+== Creating a cluster role
+
+. Export a cluster role to modify:
++
+----
+oc get clusterrole admin -o yaml > newrole.yaml
```

Comment:
No YAML, this should follow the same pattern as the "create local role" above.

Reply:
@kalexand-rh Did you get this comment ^^ just checkin'